### PR TITLE
Symantec Endpoint Protection test playbook

### PR DESCRIPTION
**Pull request by nshap**
Fixes https://github.com/demisto/server/issues/7576
Previously a POC, now a proper playbook.

### CODE REVIEW

File: Scripts/script-Sleep.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 commonfields:
   id: Sleep
-  version: -1
+  version: 7
```
Comment (Spivaka) [inline Scripts/script-Sleep.yml]: should be minus 1

File: Scripts/script-Sleep.yml [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 name: Sleep
-script: |
-  import time
-  time.sleep(int(demisto.args()['seconds']));
-type: python
+script: wait(parseInt(args.seconds));
```
Comment (Spivaka) [inline Scripts/script-Sleep.yml]: great!

File: Scripts/script-Sleep.yml [hunk 3]
```diff
@@ -16,5 +14,4 @@ args:
   default: true
   description: Amount of seconds to sleep
 scripttarget: 0
-dependson: {}
-timeout: 6000s
+timeout: 1h40m0s
```
Comment (Spivaka) [inline Scripts/script-Sleep.yml]: why?

Comment (nshap) [inline Scripts/script-Sleep.yml]: It was 6000s in the Python version (I don't know why - I think it's arbitrary), which is 1 hour and 40 minutes.

**Pull request merged**